judge/management/commands/generate_api_token.py

```diff
@@ -0,0 +1,17 @@
+import pyotp
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from judge.models import Profile
+
+
+class Command(BaseCommand):
+    help = 'generates or regenerates API token for user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('name', help='username')
+
+    def handle(self, *args, **options):
+        token = pyotp.random_base32(length=32).lower()
```

Comment:
Aren't you supposed to like... tell people what the API token is? Or are they supposed to look it up?

Reply:
Now that a switch has been done to hashing the tokens, the functionality of this command must change as well. Should it output the token created for the user, and if so, in what output format (json, etc.)?

Reply:
It definitely needs to output the token, and it can just display it in plain text.

Reply:
Fixed